honeybee_radiance/properties.py

```diff
@@ -61,7 +62,7 @@ def modifier_dir(self):
 
     @modifier_dir.setter
     def modifier_dir(self, value):
-        if hasattr(value, 'can_be_modifier') and value.can_be_modifier:
+        if isinstance(value, Primitive) and value.is_modifier:
```

Comment:
Shouldn't we be able to just do `isinstance(value, Modifier)` in such cases now that it has its own type?

Reply:
Yes, we could but there's an open question as to whether we want to allow people to assign a `Primitive` here that is of the `Primitive` base class and isn't of a specific modifier class. You can see that [the `is_modifier` property of the Primitive class](https://github.com/ladybug-tools/honeybee-radiance/pull/31/files#diff-7e8803b3f5c02bf9bde1c8b715f1ee34R309-R315) is good enough on its own to tell you whether Radiance will interpret the primitive as a modifier. But I don't know if this flexibility is useful at all and, if you think it's better to enforce that the object be of the `Modifier` subclass, I will do that.

In any case, I might leave this as-is for now since the next PR is the one where I'll be overwriting all of the extension properties.